drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -165,6 +165,11 @@ matterManufacturer:
     vendorId: 0x1339
     productId: 0x007B
     deviceProfileName: light-color-level-2000K-7000K
+  - id: "4921/107"
+    deviceLabel: Cync Reveal Full Color A19
+    vendorId: 0x1339
+    productId: 0x006D
```

Comment:
Is this supposed to be "Cync Reveal Full Color A21" instead of A19? It looks like that is what this productID is for according to the DCL, and the ticket mentions A21 as well (in the title of the ticket at least)

Reply:
Great Catch Nick! I missed that. I will look to confirm....

Reply:
Fixed!

Reply:
Great! I will approve